impl_lite/src/main/java/io/opencensus/trace/TraceComponentImplLite.java

```diff
@@ -13,10 +13,10 @@
 
 package io.opencensus.trace;
 
-import io.opencensus.impl.common.MillisClock;
-import io.opencensus.impl.internal.SimpleEventQueue;
-import io.opencensus.impl.trace.TraceComponentImplBase;
-import io.opencensus.impl.trace.internal.RandomHandler.SecureRandomHandler;
+import io.opencensus.implcore.common.MillisClock;
+import io.opencensus.implcore.internal.SimpleEventQueue;
+import io.opencensus.implcore.trace.TraceComponentImplBase;
+import io.opencensus.implcore.trace.internal.RandomHandler.SecureRandomHandler;
 
 /** Android-compatible implementation of the {@link TraceComponent}. */
 // TraceComponentImplLite was moved to io.opencensus.impl.trace. This class exists for backwards
```

Comment:
We still need this class for backwards compatibility with opencensus-api 0.5, though I forgot to update the package name in the comment.